neurom/analysis/morphtree.py

```diff
@@ -264,6 +264,40 @@ def trunk_length(tree):
         return 0.0
 
 
+def trunk_vector(tree, soma):
+    '''Vector of trunk defined as (initial tree point - soma center)
+       of the tree.
+    '''
+    return mm.vector(tree.value, soma.center)
+
+
+def trunk_elevation(tree, soma):
```

Comment:
(The following may be a question of taste) Actually, I use trunk elevation and trunk azimuth for the orientation of the first segment, and initial elevation and initial azimuth for the positioning of the first cross-section of a neurite.